src/main/java/gov/nasa/jpl/mbee/controllers/PresentationController.java

```diff
@@ -39,47 +40,63 @@
         String projectId, String refId, String presentationId,
             @QueryValue("alf_ticket") Optional<String> ticket) {
 
-        ApiClient client;
-        ElementApi apiInstance;
         Map<String, Object> response = new HashMap<>();
-
         try {
-            client = Utils.createClient(ticket, auth, url);
-            apiInstance = new ElementApi();
+            ApiClient client = Utils.createClient(ticket, auth, url);
+            ElementApi apiInstance = new ElementApi();
             apiInstance.setApiClient(client);
             Element pe = Utils.getElement(apiInstance, projectId, refId, presentationId, null);
             PresentationElement responsePE = Utils.buildResponsePe(pe);
             response.put("element", responsePE);
+            response.put("status", "ok");
```

Comment:
change the response to array of "elements" to be consistent

Reply:
what do you want to return? the swagger doesn't have anything for these endpoints... should i just remove the "status"?